src/js/rx/containers/Active.jsx

```diff
@@ -28,20 +33,35 @@ class Active extends React.Component {
     }
   }
 
-  componentDidUpdate() {
-    const newSort = this.props.location.query.sort;
-    const oldSort = this.props.sort;
-
-    if (newSort !== oldSort) {
-      this.props.sortPrescriptions(newSort);
-    }
-  }
-
-  handleSort(sort) {
+  handleSort(sortKey, order) {
+    const sortParam = order === 'DESC' ? sortKey : `-${sortKey}`;
```

Comment:
Actually, I take that back on the edit. (Reading old code is hard...) It looks like the handler should be passing the order that is being requested, so 'DESC' should correspond to the negative key. So `prescriptionName` is 'ASC' while `-prescriptionName` is 'DESC'.